Project.toml

```
@@ -45,7 +45,7 @@ EllipticalSliceSampling = "0.2"
 ForwardDiff = "0.10.3"
 Libtask = "0.3.1"
 LogDensityProblems = "^0.9, 0.10"
-MCMCChains = "3"
+MCMCChains = "^3.0.7"
```

Since the caret specifier is the default, it's the same as
```suggestion
MCMCChains = "3.0.7"
```